modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js

```diff
@@ -49,6 +49,8 @@ AUI.add(
 				instance._renderOperator(index, condition, contentBox.one('.condition-operator-' + index));
 				instance._renderSecondOperandType(index, condition, contentBox.one('.condition-the-' + index));
 				instance._renderSecondOperandInput(index, condition, contentBox.one('.condition-type-value-' + index));
+				instance._renderSecondOperandNumeric(index, condition, contentBox.one('.condition-type-value-' + index));
+				instance._renderSecondOperandDate(index, condition, contentBox.one('.condition-type-value-' + index));
```

Comment:
Hey @alinedoleron can you re-order this functions calls? `_renderSecondOperandNumeric` and `_renderSecondOperandDate`

Reply:
ok